devel/HaXml/Portfile

```diff
@@ -21,10 +21,11 @@ homepage            https://projects.haskell.org/${name}
 
 master_sites        https://hackage.haskell.org/package/${name}-${version}
 
-checksums           sha1    f44cda704057f27b0bd5cc8125406a8cd3d2013b \
-                    rmd160  b32efb8c2e04e876df4a463d68bdea23af1361d0 \
-                    sha256  01bbc66cd260286bb7b51f94f03854eae56797d0137a5b89d2a9f4769d3b4245 \
-                    size    147629
+checksums           sha1    ff4b27d6319228a7eee8803ecdb139bc3de1e7c4 \
```

Comment:
`sha1` is no longer a recommended checksum type - we should drop this.

Reply:
Thank you. Done.